common/app/views/support/Basher.scala

```diff
@@ -0,0 +1,15 @@
+package views.support
+
+object Basher {
+    def fromId(id: String) = {
+        if (id == "environment/series/keep-it-in-the-ground") {
+            KeepItInTheGround
+        } else {
+            None
+        }
+    }
+}
```

Comment:
This function returns a type of `Any` which is bad because you lose type safety. You should be explicit about what type you are returning:

``` scala
def fromId(id: String): Option[Basher] = {
```

It looks like you want to return an option? So either `None` or `Some(value)`?